app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -68,74 +62,92 @@ import org.mozilla.fenix.wallpapers.Wallpaper
+                        selectedWallpaper = selectedWallpaper,
+                        onSelectWallpaper = { updatedWallpaper ->
+                            onSelectWallpaper(updatedWallpaper)
+                        },
+                    )
+                }
+            }
         }
     }
 }
 
 @Composable
-private fun WallpaperSnackbar(
-    onViewWallpaper: () -> Unit,
+private fun WallpaperGroupHeading(
+    onLearnMoreClick: (String) -> Unit,
+    collection: Wallpaper.Collection,
 ) {
-    Snackbar(
-        modifier = Modifier
-            .padding(8.dp)
-            .heightIn(min = 48.dp),
-        backgroundColor = FirefoxTheme.colors.actionPrimary,
-        content = {
+    Column(
+        modifier = Modifier.padding(
+            top = 16.dp,
+            start = 18.dp,
+            end = 18.dp,
+            bottom = 12.dp,
+        ),
+    ) {
+        if (collection.name != "classic-firefox") {
             Text(
-                modifier = Modifier.padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
-                text = stringResource(R.string.wallpaper_updated_snackbar_message),
-                textAlign = TextAlign.Start,
-                color = FirefoxTheme.colors.textOnColorPrimary,
-                overflow = TextOverflow.Ellipsis,
-                maxLines = 2,
-                style = FirefoxTheme.typography.headline7,
+                text = collection.heading ?: "Limited edition",
+                modifier = Modifier.padding(bottom = 2.dp),
+                color = FirefoxTheme.colors.textSecondary,
+                style = FirefoxTheme.typography.subtitle2,
             )
-        },
-        action = {
-            TextButton(
-                text = stringResource(R.string.wallpaper_updated_snackbar_action),
-                onClick = onViewWallpaper,
-                modifier = Modifier.padding(all = 8.dp),
-                textColor = FirefoxTheme.colors.textOnColorPrimary,
+
+            Row(modifier = Modifier.fillMaxWidth()) {
+                collection.description?.let {
+                    Text(
+                        text = "$it. ",
+                        textAlign = TextAlign.Start,
+                        color = FirefoxTheme.colors.textSecondary,
+                        style = FirefoxTheme.typography.caption,
+                    )
+                }
+                collection.learnMoreUrl?.let {
+                    Text(
+                        text = stringResource(R.string.wallpaper_learn_more),
+                        color = FirefoxTheme.colors.textSecondary,
+                        modifier = Modifier.clickable { onLearnMoreClick(collection.learnMoreUrl) },
+                        style = FirefoxTheme.typography.caption.copy(
+                            textDecoration = TextDecoration.Underline,
+                        ),
+                    )
+                }
+            }
+        } else {
+            Text(
+                text = "Classic Firefox",
```

Comment:
This should probably be a string resource, since it doesn't need to be dynamic and can be translated.